docs/docs/en/guide/parameter/context.md

```diff
@@ -64,3 +64,25 @@ For example, in the figure below:
 When the log detects the `${setValue(key=value1)}` format in the shell node definition, it will assign value1 to the key, and downstream nodes can use the variable key directly. Similarly, you can find the corresponding node instance on the [Workflow Instance] page to see the value of the variable.
 
 ![png10](/img/globalParam/image-20210723102522383.png)
+
+### HTTP
+
+Step 1: Drag an HTTP task, fill IN body for KEY, select OUT for IN/OUT, select VARCHAR for output data type, always select VARCHAR, nothing else.
+
+<img src="/img/httpParam/1-1.png" alt="1-1" style="zoom:50%;" />
```

Comment:
* use markdown syntax when img using, ex `[httpParam1](/img/httpParam1-1.png)`. same as other imgs
* please use the English version of screenshot
* change the img file to more meaningful name instead of `1-1.png` and `1-2.png`, maybe you could directly using `/img/httpparam-1.png` and so on

Reply:
It has been resolved and the code has been committed